prime-dotnet-webapi-tests/ModelFactories/EnrolleeFactory.cs

```diff
@@ -56,13 +56,13 @@ public EnrolleeFactory()
             RuleFor(x => x.EnrolleeProfileVersions, f => null);
             Ignore(x => x.AccessTerms); // Awaiting finialization of access terms
 
-            RuleSet("status.under_review", (set) =>
+            RuleSet("status.submitted", (set) =>
```

Comment:
Why do we have submitted instead of under review here?

Reply:
This creates enrollees with a certain set of statuses, rather than just one status. In this case, it's "active" then "under review". I called it "submitted" to try to distinguish it from the name of a status, but its not a perfect naming.